util/src/main/java/org/datacommons/util/SummaryReportGenerator.java

```diff
@@ -95,15 +95,37 @@ public static String constructSVGChartFromTimeSeries(TimeSeries timeSeries) {
       if (timeSeries.isEmpty()) return "";
       dataset.addSeries(timeSeries);
       // create the time series chart with default settings
-      JFreeChart chart = ChartFactory.createTimeSeriesChart("", "", "", dataset);
+      JFreeChart chart = ChartFactory.createTimeSeriesChart("", "", "", null);
       XYPlot plot = chart.getXYPlot();
-      // create and use a renderer to draw each data point on the time series chart as a diamond and
-      // remove the legend
-      XYItemRenderer renderer = new XYLineAndShapeRenderer(true, true);
-      renderer.setSeriesShape(0, ShapeUtils.createDiamond(5));
-      renderer.setSeriesVisibleInLegend(0, false);
-      plot.setRenderer(renderer);
-      // change the background color of the chart to be white
+      // create and use separate renderers for the line and the shapes to be able
```

Comment:
nit: Maybe this could be in a helper function? setRenderers(plot)

Reply:
My thought is to that this is perhaps for future development around plotting; because this function is the only place in the codebase we deal with SVG/charting so far.